.travis.yml

```diff
@@ -1,12 +1,12 @@
 language: python
-dist: trusty
+dist: xenial
 sudo: false
 cache:
   pip: true
 
 python:
-  - "3.5"
```

Comment:
Is it possible to retain 3.5 for testing?

Reply:
updated